src/client.ts

```diff
@@ -3398,20 +3399,45 @@ export class MatrixClient extends EventEmitter {
         content: IContent,
         txnId?: string,
         callback?: Callback,
+    );
+    public sendEvent(
+        roomId: string,
+        threadId: string | null,
+        eventType: string,
+        content: IContent,
+        txnId?: string,
+        callback?: Callback,
+    )
+    public sendEvent(
+        roomId: string,
+        threadId: string | null,
+        eventType: string | IContent,
+        content: IContent | string,
+        txnId?: string | Callback,
+        callback?: Callback,
     ): Promise<ISendEventResponse> {
-        return this.sendCompleteEvent(roomId, { type: eventType, content }, txnId, callback);
+        if (!threadId?.startsWith("$") && threadId !== null) {
```

Comment:
I'm really not a fan of string matching these things - can we pass an object of some kind for `instanceof` checks instead? The types will be a lot easier to work with.

Reply:
They could be, but the reason why I opted out from that was to keep it simple to send an event. 
Not requiring a developer to instanciate 5 different models before they can even send something

The way the app is architectured too, the composer is not really aware of the `Thread` model either, but is aware of a relation to attach to all event it emits out.
I decided to go with that solution to keep the door open to more changes in the future and not tie down the composer to the current model of threads